File: java/Fib.java
class Fib
{
    static long fib(long x)
    {
        if (x <= 1) {
            return 1;
        } else {
            return fib(x-1) + fib(x-2);
        }
    }

    public static void main(String []args)
    {
        long x = Long.parseLong(args[0], 10);
        System.out.println(fib(x));
    }
};

File: go/fib.go
package main

import (
	"fmt"
	"os"
	"strconv"
)

func fib(x uint64) uint64 {
	if x <= 1 {
		return 1
	}
	return fib(x-1) + fib(x-2)
}

func main() {
	x, err := strconv.ParseUint(os.Args[1], 10, 64)
	if err != nil {
		panic(err)
	}
	fmt.Println(fib(x))
}
